### PR TITLE
Add conditions page

### PR DESCRIPTION
**Pull request by stimms**

### CODE REVIEW

File: .cr/Settings.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Containers>
+  <Container Key="942383484">
+    <Sessions>
+      <Item Id="MainSession">
+        <Filter Type="DevExpress.CodeRush.TestRunner.Data.Filters.TestFilterAnd" Assembly="DevExpress.CodeRush.TestRunner.Data, Version=16.1.8.0, Culture=neutral, PublicKeyToken=79868b8147b5eae4">
```
Comment (MisterJames) [inline .cr/Settings.xml]: code rush?

File: src/AlpineSkiHouse.Web/wwwroot/_references.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
 ﻿/// <autosync enabled="true" />
+/// <reference path="../bower_components/bootstrap/dist/js/bootstrap.js" />
```
Comment (MisterJames) [inline src/AlpineSkiHouse.Web/wwwroot/_references.js:2]: we should probably ignore this file

File: src/AlpineSkiHouse.WebJobs/project.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   },
 
   "dependencies": {
-    "ImageProcessorCore": "1.0.0-alpha1058",
+    "ImageProcessorCore": "1.0.0-alpha1031",
```
Comment (MisterJames) [inline src/AlpineSkiHouse.WebJobs/project.json:8]: Why the regress?

File: .cr/Settings.xml (hunk 1 above)
Comment (stimms) [reply .cr/Settings.xml]: Apparently. Should probably be in the ignore.

File: src/AlpineSkiHouse.Web/wwwroot/_references.js (hunk 2 above)
Comment (stimms) [reply src/AlpineSkiHouse.Web/wwwroot/_references.js:2]: I thought that this file is used for typescript, I don't think including it is a problem.

File: src/AlpineSkiHouse.WebJobs/project.json (hunk 3 above)
Comment (stimms) [reply src/AlpineSkiHouse.WebJobs/project.json:8]: That is the latest version I could see on nuget

File: src/AlpineSkiHouse.Web/wwwroot/_references.js (hunk 2 above)
Comment (dpaquette) [reply src/AlpineSkiHouse.Web/wwwroot/_references.js:2]: yeah it's fine to include this

**Comment by dpaquette** [comment]
Looks good to me :shipit:

**Pull request merged**